src/cli/tsconfig-for-init.json

```diff
@@ -1,22 +1,22 @@
 {
   "compilerOptions": {
+    // Enable latest features
     "lib": ["ESNext"],
     "target": "ESNext",
     "module": "ESNext",
     "moduleDetection": "force",
-    "jsx": "react-jsx",
-    "allowJs": true,
+    "jsx": "react-jsx", // support JSX
```

Comment:
is it not obvious setting the tsconfig option `jsx` would enable jsx support.

i dont think this comment adds any value